app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2362,6 +2371,98 @@ class BrowserTabFragment :
         experimentCta.hideOnboardingCta(binding)
     }
 
+    private fun configureWebViewForBlobDownload(webView: DuckDuckGoWebView) {
+        lifecycleScope.launch(dispatchers.main()) {
+            if (isBlobDownloadWebViewFeatureEnabled(webView)) {
+                val script = blobDownloadScript()
+                WebViewCompat.addDocumentStartJavaScript(webView, script, setOf("*"))
```

Comment:
can you take care of the lint warning on this line since your other check already guarantees it's supported?

Reply:
I don't get a warning in the IDE for this.

Reply:
I'm almost certain I did see it in the IDE, as that is what made me think of it. Not seeing it now. 😕 . Feel free to ignore.